jwst/assign_wcs/nirspec.py

```diff
@@ -444,7 +453,9 @@ def get_open_msa_slits(msa_file, msa_metadata_id, slit_y_range=[-.55, .55]):
         ('background', 'S1'),
         ('shutter_state', 'S6'),
         ('estimated_source_in_shutter_x', '>f4'),
-        ('estimated_source_in_shutter_y', '>f4')])
+        ('estimated_source_in_shutter_y', '>f4')
```

Comment:
This probably should have a trailing comma, though I realize it's just a doc string.